### ISSUE TITLE
Is sortable multi-select supported?

### ISSUE THREAD

**Issue by linhpham199** [based-ghost/react-functional-select#24]
Hi,
For a huge number of options in the select list, I think a sortable feature would be useful. As I can see from the docs that the library doesn't support this feature itself; however, I wonder if it is possible to use this library with other library that supports sortable, for example react-sortable-hoc? Or if there is other possible implementation for sortable feature, I'd appreciate if you could show me.
Thanks in advance!

**Comment by based-ghost** [comment]
Hey @linhpham199 - I agree, that would be useful for multi-select scenarios. To clarify you mean sortable in the sense that the selected value tags are able to be re-arranged (e.g. drag-and-drop functionality), correct? I'll have to do a little investigation to see the optimal way to implement this, as it has the possibility to introduce significant bundle size if done quickly.

If you find any examples of packages that achieve something like this in a low-code, and/or optional dependency way let me know!

Thanks.

**Comment by linhpham199** [comment]
Yes, drag-and-drop functionality is exactly what I meant, so the users can change the order of selected values in the list as they like.

And definitely if I find any good packages, I'll inform you. So far I've only seen [react-sortable-hoc combining react-select](https://react-select.com/advanced#sortable-multiselect) to support drag-and-drop, and in this case react-sortable-hoc is optional dependency.
